client/state/plugins/installed/actions.js

```diff
@@ -67,8 +69,57 @@ export function activatePlugin( siteId, plugin ) {
 		};
 		dispatch( { ...defaultAction, type: PLUGIN_ACTIVATE_REQUEST } );
 
+		// TODO: Remove when this flux action is completely reduxified
+		Dispatcher.handleViewAction( {
+			type: 'ACTIVATE_PLUGIN',
+			action: 'ACTIVATE_PLUGIN',
+			site,
+			plugin,
+		} );
+
+		const afterActivationCallback = ( data, error ) => {
```

Comment:
Shall we stick to the node style convention that in callbacks the error param comes first?

Reply:
Makes sense. Addressed in https://github.com/Automattic/wp-calypso/pull/47711/commits/40e86cf6e9780f8333b8a5d4403237b7240b26ad